src/test/java/org/traccar/protocol/Gt06ProtocolDecoderTest.java

```diff
@@ -300,6 +300,26 @@ public void testDecode() throws Exception {
         verifyNull(decoder, binary(
                 "7878058A000688290D0A"));
 
+        verifyAttribute(decoder, binary(
+                "78780c95130a0209321c90000112800d0a"),
+                "alarm", "hardAcceleration");
```

Comment:
Please use constants from `Position`. Also, we probably don't need a test for each single alarm. 1-2 samples would be more than enough.

Reply:
OK, no prob.